pkg/alerts/nrql_conditions.go

```diff
@@ -731,6 +732,7 @@ const (
       closeViolationsOnExpiration
       expirationDuration
       openViolationOnExpiration
+			ignoreOnExpectedTermination
```

Comment:
Is this indentation weird or is it just on my screen?

Reply:
Hey, it seemed to me like this was actually unindented in the code, which is why I tried fixing it; and it turns out it actually is something weird with GitHub. Apologies, please ignore my latest commit `38a14d7`

(here's what it looks like after the indent was fixed, but GitHub's diff still shows an extra indent).
<img width="365" alt="image" src="https://github.com/user-attachments/assets/485a2330-62dd-46bb-8872-6e488ce2b238">